app/controllers/application_controller.rb

```diff
@@ -52,6 +44,6 @@ def set_locale
   end
 
   def store_request_id
-    RequestStore.store[:request_id] = request.uuid
+    Instrumentation.append_to_log(request_id: request.uuid)
```

Comment:
Aw, nuts.  That makes it way more magic than I was thinking it was...It'll think about it and see what I can come up with.